lib/parallel_tests/cli.rb

```diff
@@ -28,7 +28,8 @@ def execute_in_parallel(items, num_processes, options)
       Tempfile.open 'parallel_tests-lock' do |lock|
         progress_indicator = simulate_output_for_ci if options[:serialize_stdout]
 
-        Parallel.map(items, :in_threads => num_processes) do |item|
+        ParallelTests.pids # setup the class vars here so we dont initialize in the threads
```

Comment:
hmm got a mutex for that ?

Reply:
... but getting the ENV var set is good ...

Reply:
Not sure if the mutex would help here. It needs to be called so the `@pids` var gets set before the threads start. Otherwise, each thread would have it's own new instance of Pids. Let me know if you have any ideas or better way to do this.